bundle/config/mutator/resolve_schema_dependency.go

```diff
@@ -0,0 +1,80 @@
+package mutator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/libs/diag"
+)
+
+type resolveSchemeDependency struct{}
```

Comment:
```suggestion
type resolveSchemaDependency struct{}
```

Reply:
It's not really resolving a dependency, it's capturing the dependency (like the PR title).

Reply:
done